example.js

```diff
@@ -3,11 +3,11 @@
 import createClient, {SERVER_TIMING, CACHE} from './index.js'
 import {inspect} from 'util'
 
-const vbbClient = createClient('https://v5.vbb.transport.rest', {
+const vbbClient = createClient('https://v6.vbb.transport.rest', {
 	userAgent: 'hafas-rest-api-client example',
 })
 
-vbbClient.journeys('900000003201', '900000024101', {results: 1})
+vbbClient.journeys('900100705', '900096197', {results: 1})
```

Comment:
Let's pick `900100003` (the Alexanderplatz parent station) as an example.

With `900096197`, the VBB HAFAS reports "location missing or invalid".